### PR TITLE
chore(docs):[#223] Create concept for transfer to PCWM

### PR DESCRIPTION
**Pull request by ds-jhartmann**

### CODE REVIEW

**Review by ds-psosnowski — APPROVED** [review]
lgtm

File: docs/concept/#223-Transfer-to-PCWM/#223-Transfer-to-PCWM.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## How to migrate the Confluence pages to GitHub (Pages, Wiki, MD, issues)
+
+Only a small portion of our Confluence documentation has to be migrated to GitHub, so copy-pasting the content manually is sufficient and the fastest solution.
+
+Confluence pages which shall be moved to GitHub have to be marked as such. Everything else will not be moved.
+
+## Test management and Test Reporting (Cucumber)
+
+Cucumber Tests are were stored in and reported to Jira, using the Jira-Xray plugin.
```
Comment (mkanal) [inline docs/concept/#223-Transfer-to-PCWM/#223-Transfer-to-PCWM.md:11]:
```suggestion
AsIs:
Cucumber Tests are were stored in and reported to Jira, using the Jira-Xray plugin. 
```

File: docs/concept/#223-Transfer-to-PCWM/#223-Transfer-to-PCWM.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Test results have to be displayed directly in GitHub workflow / Pull Request. This can be done in two ways:
+
+- using the free [Cucumber Report Service](https://cucumber.io/docs/cucumber/reporting/?lang=java#cucumber-reports-service), where the results will be published to a cloud hosted service
+- Using the generated test report and upload it using a GitHub Action https://github.com/marketplace/actions/publish-test-results
+
+## Replacing the CatenaX-NG environment
+
+The CatenaX-NG ArgoCD environments were used to execute Integration- and E2E-Tests. After the end of the Consortia phase, they are no longer available and have to be replaced by an open solution.
+
+The proposed solution is a helm chart with all required dependencies, including EDC Consumer, EDC Provider, MIW,
```
Comment (mkanal) [inline docs/concept/#223-Transfer-to-PCWM/#223-Transfer-to-PCWM.md:24]:
```suggestion
ToBe:
The proposed solution is a helm chart with all required dependencies, including EDC Consumer, EDC Provider, MIW,
```

File: docs/concept/#223-Transfer-to-PCWM/#223-Transfer-to-PCWM.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Cucumber Tests are were stored in and reported to Jira, using the Jira-Xray plugin. 
+
+Feature files can be exported and directly stored in GitHub.
+
+Test results have to be displayed directly in GitHub workflow / Pull Request. This can be done in two ways:
+
+- using the free [Cucumber Report Service](https://cucumber.io/docs/cucumber/reporting/?lang=java#cucumber-reports-service), where the results will be published to a cloud hosted service
+- Using the generated test report and upload it using a GitHub Action https://github.com/marketplace/actions/publish-test-results
+
```
[the remark this answers is not in the document]
Comment (ds-jhartmann) [reply docs/concept/#223-Transfer-to-PCWM/#223-Transfer-to-PCWM.md:19]: Cucumer is published under the MIT license and therefore free to use. Same goes for the Cucumber Report Service.
The alternative via test report GitHub Action is also open source and free.

**Review by mkanal — APPROVED** [review]

**Pull request merged**